src/exec/storage.go

```diff
@@ -51,6 +51,16 @@ func (s storage) Run(config config.Config) bool {
 		return false
 	}
 
+	if err := s.createPasswd(config); err != nil {
+		s.Logger.Crit("failed to create users/groups: %v", err)
+		return false
+	}
+
+	if err := s.createFilesystemsFiles(config); err != nil {
```

Comment:
Well, they're certainly related.

The creation of users/groups needs to happen before we create files which may be owned by those users, but after we create the filesystem the pwdb is stored on.